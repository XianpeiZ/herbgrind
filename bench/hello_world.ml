let say_hello _ =
  let rand = Random.float(1.0) in
  let x = sqrt(rand) in
  Printf.printf "Hello world %e!\n" x;;

say_hello () ;;
