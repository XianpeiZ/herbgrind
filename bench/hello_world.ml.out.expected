(output
  (function "camlHello_world__say_hello_1199")
  (filename "hello_world.ml")
  (line-num 3)
  (instr-addr 41C4AB)
  (avg-error 0.000000)
  (max-error 0.000000)
  (num-calls 2)
  (influences
    (
    )
  )
)
